src/thor/bidirectional_astar.cc

```diff
@@ -336,11 +338,13 @@ inline bool BidirectionalAStar::ExpandInner(baldr::GraphReader& graphreader,
     }
     edgelabels_reverse_.emplace_back(pred_idx, meta.edge_id, opp_edge_id, meta.edge, newcost,
                                      sortcost, dist, mode_, transition_cost, thru,
-                                     (pred.closure_pruning() || !costing_->IsClosed(meta.edge, tile)),
+                                     (pred.closure_pruning() || !costing_->IsClosed(opp_edge, tile)),
```

Comment:
this was bugging me before: why would we look at the traversed edge for closures when we're actually interested in the opposing for reverse search? the same goes for `not_thru` further above, but that might actually not be a directed property, not sure..

Reply:
same in all other algorithms. which makes me think this is not a copy/paste fail but intentional. I just don't really get it I guess..

Reply:
revert for now and leave todo